src/DependencyInjection/Compiler/EventHandlerCompilerPass.php

```diff
@@ -27,9 +28,7 @@ public function process(ContainerBuilder $container)
 
         foreach ($taggedServices as $id => $tags) {
             foreach ($tags as $attributes) {
-                if (array_key_exists('service', $attributes)
-                    && false !== strstr($attributes['event'], 'guzzle_bundle')) {
-
+                if (isset($attributes['service']) && in_array($attributes['event'], GuzzleEvents::EVENTS, true)) {
```

Comment:
@florianpreusner here we had bug. When we migrated to flex the events were renamed. For example  `guzzle_bundle.pre_transaction` became `eight_points_guzzle.pre_transaction` but this verification was not adapted. I fixed it and added more strict verification.

Reply:
good shot! 👍